src/main/java/com/cgm/infolab/RunAfterStartup.java

```diff
@@ -63,8 +66,11 @@ public void addComponentsToDb() {
         if(Arrays.asList(env.getActiveProfiles()).contains("dev")){
             saveRooms(TEST_ROOMS);
             saveUsers(TEST_USERS);
-        }
 
+            System.out.println(
+                    "URL to login as lorenzo through jwt: http://localhost:%d/?access_token=%s"
+                            .formatted(port, new TestJwtHelper().generateToken("infolab", "lorenzo").getTokenValue()));
```

Comment:
in `generateToken` ci mettiamo un'opzione in più per aumentare il tempo di scadenza? qui metterei più di un minuto... aumenterei almeno a un'ora ma anche di più. Fai un overload del metodo e metti il tempo come parametro opzionale, di modo che per i test lasciamo un valore di default